tests/standalone/run_all.py

```diff
@@ -264,6 +264,15 @@
     "MFCM140U.DLL",
 )
 
+# finds the special comment in each test module and returns the import module needed for each test
+# special comments are the the following format:
+# "# nuitka-skip-test-unless: module"
+def find_required_module(filename):
```

Comment:
Ought to be named `findRequiredModules`, we use came case for methods.

And yes, I think you should support plural to be safe, i.e. yield module names.